package.json

```diff
@@ -1,19 +1,43 @@
 {
-  "//": "limitations under the License.",
+  "//-01": "Licensed to the Apache Software Foundation (ASF) under one or more",
+  "//-02": "contributor license agreements.  See the NOTICE file distributed with",
+  "//-03": "this work for additional information regarding copyright ownership.",
+  "//-04": "The ASF licenses this file to You under the Apache License, Version 2.0",
+  "//-05": "(the \"License\"); you may not use this file except in compliance with",
+  "//-06": "the License.  You may obtain a copy of the License at",
+  "//-07": "",
+  "//-08": "http://www.apache.org/licenses/LICENSE-2.0",
+  "//-09": "",
+  "//-10": "Unless required by applicable law or agreed to in writing, software",
+  "//-11": "distributed under the License is distributed on an \"AS IS\" BASIS,",
+  "//-12": "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+  "//-13": "See the License for the specific language governing permissions and",
+  "//-14": "limitations under the License.",
   "name": "ngx-webpack-seed",
   "version": "0.0.0",
   "scripts": {
-    "start": "./node_modules/http-server/bin/http-server .",
-    "watch": "./node_modules/grunt-cli/bin/grunt",
-    "build:theme": "./node_modules/grunt-cli/bin/grunt compile-web-ui-styles",
-    "build:webapp:dev": "./node_modules/grunt-cli/bin/grunt dev-bundle-web-ui",
-    "build:webapp:prod": "./node_modules/grunt-cli/bin/grunt prod-bundle-web-ui",
-    "clean:install": "bash ./scripts/clean-install",
-    "clean:install:skipTests": "bash ./scripts/clean-install-skipTests",
+    "start": "npm run watch",
+    "clean-and-copy": "node scripts/clean.js && node scripts/copy-files.js",
+    "build:webapp:bundle:development": "npm run webpack",
+    "build:webapp:bundle:production": "npm run webpack:production",
+    "clean:all": "node scripts/clean.js && node scripts/clean-dev.js",
+    "clean:install": "npm run clean-and-copy && cd ./target/frontend-working-directory && npm install && npm test && npm run build:webapp:bundle:production",
```

Comment:
Wouldn't we want to run `npm run test:parallel` here?

Reply:
Yeah, I can change that. For the seed project it is actually slower since it has the overhead of 4 chrome's to run like 3 tests. But i see your point.